### DISCUSSION TITLE
Question about webview

### DISCUSSION THREAD

**Discussion by heberjeur** [Slion/Fulguris#602] [General]
Hi, I want to know if any sensitive data can be transmitted to Google via WebView. Also, are browsers based on Firefox Geckoview more secure because they don't use WebView?

**Comment by marendowski** [comment]
I think that security-oriented WebViews (Bromite, Vanadium, Mulch etc.) or GeckoView without the telemetry and with other features is something I want and need to install because I use older systems with WebView too old to even render websites. You need to do more research yourself to see if they are more secure.

**Comment by Slion** [comment]
A WebView browser when configured properly should be as secure as any Chromium bases browser.
Though there are indeed concerns, see notably #585.

**Comment by ghost** [comment]
I was just reading today about this very thing on GrapheneOS and their vanadium browser vs webview.

Webview lacks certain privacy/security features, at this point in time, according to GrapheneOS. Such as per site sandboxing.  You can read more here: https://grapheneos.org/usage

I've also heard this same thing many times from Stoutner, the dev of Privacy Browser. 

However, imo, there are trade offs any way you go. I've noticed that many browser's "user data" continually grows at a rather alarming rate — what's being collected? I make no changes, just visit a few sites and the user data increases (not talking about cache). Only Stoutner's Privacy Browser stays at a very low user data when you clear out data from the browser, all others I've checked continually increase 🤷🏻‍♂️

Second is how you use the browser, forced always incognito mode or regular — you'll have high privacy and security using a browser that's always operation in incognito especially if you don't use many tabs at the same time = everything is always being cleared. 

Just some thoughts.
